resources/todomvc/architecture-examples/angular/dist/index.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html lang="en" data-framework="angular" data-version="14.2.11" data-features="">
-    <head>
+    <head><base href="/resources/todomvc/architecture-examples/angular/dist/">
```

Comment:
Please don't do this. This will break the ability to run Speedometer inside another directory, which is a required setup for both browserbench.org as well as WebKit's CI.

Reply:
This is the output of the npm run build command, am I missing some config? Should I just manually delete the line?

Reply:
If so, please manually delete the line.

Reply:
Yes, this is created in the build step and there have been manual fixups (i.e. https://github.com/WebKit/Speedometer/pull/161). Ideally we could change some build option or add some auto fixup, but it's out of scope for this PR